website/content/en/docs/getting-started/_index.md

```diff
@@ -1,21 +1,86 @@
+1. To install ClusterLink on Linux or Mac, use the installation script:
+
+       $ curl -L <https://Clusterlink.net/install> | sh -
+
+1. Move to the ClusterLink project folder:
+
+        cd clusterlink
+
+1. Export the ClusterLink CLI to your path:
+
+        export PATH=$PWD/bin:$PATH
+
+1. Check the installation by running the command:
+
+        clusterlink version
 
 ## Setup
 
-<!-- Is there any initial setup users need to do after installation to try your project? -->
+To set up ClusterLink on a Kubernetes cluster, follow these steps:
+
+1. Create fabric certificates.
+
+    The ClusterLink Fabric is defined as all Kubernetes clusters (sites) that install ClusterLink gateways and can share services between the clusters, enabling communication among those services.
+    First, create the fabric Certificate Authority (CA):
+
+        clusterlink create fabric --name <fabric_name>
+
+    This command will create the CA files `<fabric_name>.cert` and `<fabric_name>.key` in the current folder.
+
+2. Create site certificates.
+
+    Create a site (cluster) certificate:
+
+        clusterlink create site --name <site_name> --fabric <fabric_name>
+
+    This command will create the CA files `<site_name>.cert` and `<site_name>.key` in a  <site_name> folder.
+
+3. Install ClusterLink deployment operator.
+
+    To install ClusterLink on the site, first, install the ClusterLink deployment operator.
+
+        clusterlink site init
+
+    This command will deploy the ClusterLink deployment operator on the `clusterlink-operator` namespace and convert the site certificates to secrets in the namespace.
+    The command assumes that kubectl is set to the correct site (Kubernetes cluster) and that the certificates were created in the local folder. If they were not, use the flag -f 
+    <certificate_folder>.
+
+4. Deploy clusterlink CRD instance.
+
+    After the operator is installed, you can deploy ClusterLink by applying the ClusterLink instance CRD:
+
+        kubectl apply -f - <<EOF
+        apiVersion: clusterlink.net/v1alpha1
+        kind: ClusterLink
+        metadata:
+        namespace: clusterlink-operator
+        name: <site_name>
+        spec:
+        ingress:
+            type: <ingress_type>
+        namespace: clusterlink-system
+        EOF
+
+    If you're using a Kind cluster, replace <ingress_type> with "NodePort". For a cluster running in a cloud environment, use "LoadBalancer" instead.
+
+    The instance CRD will create the ClusterLink gateway components in the `clusterlink-system` namespace. For more details and information about the ClusterLink instance CRD, refer to the [operator documentation](https://github.com/clusterlink-net/clusterlink/blob/main/design-proposals/project-deploymnet.md#clusterlink-crd).
```

Comment:
```suggestion
    The instance CRD will create the ClusterLink gateway components in the `clusterlink-system` namespace. 
    For more details and information about the ClusterLink instance CRD, refer to the [operator documentation](https://github.com/clusterlink-net/clusterlink/blob/main/design-proposals/project-deploymnet.md#clusterlink-crd).
```

Also, we would need to replace this with the relevant documentation link as I don't want to expose design docs to users.
This would likely be in the Reference section. Or can be added here?
We can add a Collapsible/Expandable shortcode to allow for that. See first answer [here](https://stackoverflow.com/questions/71691219/add-collapsible-section-in-hugo) for example

Reply:
The object documentation and collapsible shortcode can be added in a spearate PR